build/lib/helm_push.sh

```diff
@@ -61,9 +61,12 @@ trap "rm -f $TMPFILE" exit
 helm push ${CHART_FILE} oci://${IMAGE_REGISTRY}/${HELM_DESTINATION_OWNER} | tee ${TMPFILE}
 
 # Adds a 2nd tag to the helm chart for the bundle-release jobs.
-MANIFEST=$(aws ecr batch-get-image --repository-name ${HELM_DESTINATION_REPOSITORY} --image-ids imageTag=${SEMVER} --query images[].imageManifest --output text)
-export AWS_PAGER=""
-aws ecr put-image --repository-name ${HELM_DESTINATION_REPOSITORY} --image-tag ${SEMVER_GIT_TAG}-${LATEST_TAG}-helm --image-manifest "$MANIFEST" --image-manifest-media-type "application/vnd.oci.image.manifest.v1+json"
+if [[ ${IMAGE_REGISTRY} != *"public.ecr.aws"* ]]; then
```

Comment:
```suggestion
if [[ "${IMAGE_REGISTRY}" != *"public.ecr.aws"* ]]; then
```
Without the double quotes, if $IMAGE_REGISTRY were empty, this would error with a "unary operator expected" line.